x/emissions/types/msg_create_topic.go

```diff
@@ -37,6 +37,25 @@ func (msg *MsgCreateNewTopic) Validate() error {
 	if msg.Epsilon.Lte(alloraMath.ZeroDec()) || msg.Epsilon.IsNaN() || !msg.Epsilon.IsFinite() {
 		return errors.Wrap(sdkerrors.ErrInvalidRequest, "epsilon must be greater than 0")
 	}
+	if uint64(len(msg.Metadata)) > maxMetadataLen {
+		return errors.Wrap(sdkerrors.ErrInvalidRequest, "metadata cannot be longer than max serialized msg length")
+	}
+	// no validation on AllowNegative because either it is true or false
+	// and both are valid values
+	//	AllowNegative            bool
+
+	if !isAlloraDecBetweenZeroAndOneInclusive(msg.MeritSortitionAlpha) {
+		return errors.Wrap(sdkerrors.ErrInvalidRequest, "merit sortition alpha must be between 0 and 1 inclusive")
+	}
+	if !isAlloraDecBetweenZeroAndOneInclusive(msg.ActiveInfererQuantile) {
+		return errors.Wrap(sdkerrors.ErrInvalidRequest, "active inferer quantile must be between 0 and 1 inclusive")
+	}
+	if !isAlloraDecBetweenZeroAndOneInclusive(msg.ActiveForecasterQuantile) {
+		return errors.Wrap(sdkerrors.ErrInvalidRequest, "active forecaster quantile must be between 0 and 1 inclusive")
+	}
+	if !isAlloraDecBetweenZeroAndOneInclusive(msg.ActiveReputerQuantile) {
```

Comment:
quantiles technically within [0,1)

should see what happens in the quantile-computing function when the quantile arg = 1

Reply:
you can't have a 100th percentile / quantile?

Reply:
yeah doesn't technically exist